### ISSUE TITLE
(Feature suggestion) Make Grafana allow embedding by default

### ISSUE THREAD

**Issue by eikowagenknecht** [openhab/openhabian#1464]
## Issue information:
After installation, Grafana graphs can **not** be included in OH3 Dashboards because embedding is disallowed by default, starting with version 6.2 (mid 2019).

It is a simple config variable, but I think openhabian grafana installation should enable this by default because it's one of the main use cases to embed some Grafana graphs into other OH dashboards. I've also seen some posts in the forums where other users didn't know how to enable this and were expecting it to work out of the box.

https://grafana.com/docs/grafana/latest/administration/configuration/#allow-embedding

In short, the line allow_embedding = true is needed in grafana.ini

**Comment by holgerfriedrich** [comment]
@mstormi you can assign this to me, seems I have lost all my permissions here...

**Comment by mstormi** [comment]
> 
> 
> @mstormi you can assign this to me, seems I have lost all my permissions here...

Will check your perms
What about #1332 and #1448 could you take on those as well ?
